src/App.js

```diff
@@ -1,47 +1,79 @@
 import React from 'react';
 import './App.css';
-import ButtonLogin from './components/ButtonLogin.jsx'
+import ButtonLogin from './components/ButtonLogin.jsx';
 import Playlists from './components/Playlists.jsx';
+import AddPlaylist from './components/AddPlaylist.jsx';
+import AddPlaylistPage from './components/AddPlaylistPage.jsx';
 
 class app extends React.Component {
   constructor() {
-      super();
-    
-      this.getToken();
+    super();
+    this.state = {
+      displayAdd : false,
+    }
 
+
+  }
+
+  AddPageDisplayState() {
+    this.setState({displayAdd : !this.state.displayAdd})
```

Comment:
Utiliser la version fonctionnelle: https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous